api/src/test/mock-data/response/eng-url-entities.json

```diff
@@ -19,7 +19,9 @@
       "indocChainId": 2, 
       "mention": "Washington", 
       "normalized": "Washington", 
-      "type": "LOCATION"
+      "type": "LOCATION",
+      "dbpediaType": "test1"
```

Comment:
Can you point me to the ADM slot?

Reply:
DBpediaType extended property of Entity.  FLINX LinkAnnotator populates it.